tests/baselines/reference/functionExpressionContextualTyping1.js

```diff
@@ -0,0 +1,76 @@
+//// [functionExpressionContextualTyping1.ts]
+enum E { red, blue }
+
+var g0: (n: number, s:string) => number;
+var g: ((s: string, w: boolean) => void) | ((n: number) => number);
+var g1: ((s: string, w: boolean) => void) | ((s: string, w: number) => string);
+
+g1 = (j, m) => { } // Per spec, no contextual signature can be extracted in this case.
+g = (k, h=true) => { k.toLowerCase() };
+g = (k) => { k.toLowerCase() };
+g = (i) => {
+    i.toExponential();
+    return i;
+};  // Per spec, no contextual signature can be extracted in this case.
+
+var h: ((s: string, w: boolean) => void) | ((s: string, w: boolean) => string);
+h = (k, h) => { };
+
+var i: typeof g0 | ((n: number, s: string) => string);
```

Comment:
Oh, you mean to test this case out. At least move the declaration locally and name it something separate from `g_`